src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,170 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * The Ui class handles the user interface and interacts with the user in the Duke application.
+ */
+public class Ui {
+
+    private static final String LINE = "──────────────────────────────"
+            + "────────────────────────────────────────────";
+    private static final String LOGO = " _____   __                 _____ _           _   _           _  ___\n"
+            + "|  _\\ \\ / /                /  __ \\ |         | | | |         | ||_  |\n"
+            + "| |  \\ V /___  _   _ _ __  | /  \\/ |__   __ _| |_| |__   ___ | |_ | |\n"
+            + "| |   \\ // _ \\| | | | '__| | |   | '_ \\ / _` | __| '_ \\ / _ \\| __|| |\n"
+            + "| |   | | (_) | |_| | |    | \\__/\\ | | | (_| | |_| |_) | (_) | |_ | |\n"
+            + "| |_  \\_/\\___/ \\__,_|_|     \\____/_| |_|\\__,_|\\__|_.__/ \\___/ \\__|| |\n"
+            + "|___|                                                           |___|\n";
+    private static final String greetPhrase = LINE
```

Comment:
A final variable should be in capital case! :)

greetPhrase => GREETPHRASE

Likewise for SENDOFFPHRASE